domains/zhp.pl.d/stoleczna.js

```diff
@@ -38,6 +38,7 @@ D_EXTEND('zhp.pl',
     // Hufiec Piaseczno
     Delegation_A('piaseczno', '178.32.203.125'),
     Ms365_Subdomain('piaseczno', 'zhp.pl'),
+    Delegation_A('wataha.piaseczno', '178.32.203.125'),
```

Comment:
Nie mam uprawnień do edycji, ale wysypał się build. DNScontrol ma takie zabezpieczenie, że jak masz jakąś kropkę w adresie to on tam nie wie, czy "wataha.piaseczno" to pełna domena czy może subdomena zhp.pl (to wynika z faktu, że pod `Delegation_A` jest m.in. `CNAME('www.wataha.piaseczno', 'wataha.piaseczno')` i tam on nie wie, co z tym zrobić)

Dlatego trzeba dodać trzeci parametr, 'zhp.pl' i wtedy my to ogarnęliśmy. Być może dałoby się to zrobić lepiej, ale na razie tak robimy. Stąd niżej masz `Delegation_A('szczep40.mokotow', '109.95.159.40', 'zhp.pl'),`

```suggestion
    Delegation_A('wataha.piaseczno', '178.32.203.125', 'zhp.pl'),
```